third_party/move/move-compiler-v2/tests/checking/typing/subtype_return_invalid.exp

```diff
@@ -0,0 +1,37 @@
+
+Diagnostics:
+error: mutability mismatch (&mut != &) in name expression
```

Comment:
Again, mentioning context (fun body) here is more useful than the kind of expression (name expression).

Same below.

Reply:
or maybe "fun return value" if that's possible.

Reply:
Outdated code. Those 'in' messages are all gone because of another change you requested (rightfully), they can't be longer precise.